### PR TITLE
[WIP] [feature] implement webhooks bundle in publisher

### PR DESCRIPTION
**Pull request by ahilles107**
| Q                         | A
| ------------------------- | ---
| Bug fix?                  | no
| New feature?              | yes
| BC breaks?                | no
| Deprecations?             | no
| Tests pass?               | missing
| Changelog update required | yes
| Fixed tickets             | none
| License                   | AGPLv3

### CODE REVIEW

File: src/SWP/Bundle/WebhookBundle/composer.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  ],
+  "repositories": [
+    {
+      "type": "path",
+      "url": "../../Component/*"
+    }
+  ],
+  "require": {
+    "php": "^7.0",
+    "swp/webhook": "^1.2"
```
Comment (takeit) [inline src/SWP/Bundle/WebhookBundle/composer.json]: this bundle should require storage bundle

Comment (ahilles107) [reply src/SWP/Bundle/WebhookBundle/composer.json]: yeah, requirements and tests are WIP

File: composer.json [hunk 2]
```diff
@@ -92,7 +92,8 @@
         "doctrine/doctrine-migrations-bundle": "^1.0",
         "layershifter/tld-extract": "^1.2",
         "friendsofsymfony/elastica-bundle": "^4.0",
-        "php-amqplib/rabbitmq-bundle": "^1.13"
+        "php-amqplib/rabbitmq-bundle": "^1.13",
+        "behat-extension/doctrine-data-fixtures-extension": "^4.0"
```
Comment (takeit) [inline composer.json]: shouldn't that behat extension go to `require-dev`?

Comment (ahilles107) [reply composer.json]: it should

**Pull request merged**